src/test/scala/firrtlTests/ReplSeqMemTests.scala

```diff
@@ -672,4 +672,25 @@ circuit Top :
     )
     resAnnos should be(expected)
   }
+
+  "ReplSeqMem" should "not crash if there are aggregate registers in the design (see #2379)" in {
```

Comment:
```suggestion
  "ReplSeqMem" should "not crash if there are aggregate registers in the design that need padding (see #2379)" in {
```

Or something? Isn't this specifically checking the fact that the widths are different?

Reply:
Agree. It's not specific enough... Fixed to your language in the rebase.